RefactoringEssentials/CSharp/Converter/VisualBasicConverter.cs

```diff
@@ -176,25 +191,33 @@ static bool IsVisibility(SyntaxToken token, TokenContext context)
 				|| (context == TokenContext.VariableOrConst && token.IsKind(VBasic.SyntaxKind.ConstKeyword));
 		}
 
+		/// <remarks>
+		/// Incorrectly returns private for structs instead of public
+		/// </remarks>
```

Comment:
Did you mean to commit this? I think this is a bug that should be fixed instead.